assets/js/theme.js

```diff
@@ -1,4 +1,13 @@
 const menuItems = document.querySelectorAll('ul.topics li.isParent > a');
+const isTouchDevice = ('ontouchstart' in window);
+
+function findParent(current, expected) {
+  while (current !== expected) {
+    current = current.parentNode;
```

Comment:
Don't you need a condition to break the loop when no parent is present?
If not if expected is not in the ancestors it will loop indefinitely no?

Reply:
Besides I'm not sure I get the purpose, it seems like the function will always return `expected` in the end no? (since it stops when `current === expected` then returns it)

Reply:
> Don't you need a condition to break the loop when no parent is present?

Theoretically yes, but since this is called from a click event, by definition `current` can only be a child for `expected` or the same as `expected` (`target` is either the attached element or one of its children if you clicked on them).

> Besides I'm not sure I get the purpose, it seems like the function will always return expected in the end no?

That's exactly the purpose! I was thinking about renaming it to `getThElementTheEventWasAttachedToAndNotOneOfItsChildrenGodDammit()` but it seemed too long